### DISCUSSION TITLE
Empty variables in the general health dataset that are not in the interview instruments

### DISCUSSION THREAD

**Discussion by edambo** [brad-cannell/detect_fu_interviews_public#19] [Q&A]
The following empty variables are in the general health dataset but are not in the interview instruments:

- health_bp_systolic
- health_heart
- health_oxygen
- health_temp
- health_pain

Should these variables be kept or removed if they don't contain any values and aren't in the interview instruments?

**Comment by mbcann01** [comment]
Great question. I think we should ask Sunil about them before doing anything else. Can you take the lead on that?

**Comment by edambo** [comment]
Yes, I'll do that.
